Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -199,6 +200,11 @@ class TextureCacheBase
   TexPool::iterator FindMatchingTextureFromPool(const TCacheEntryConfig& config);
   TexAddrCache::iterator GetTexCacheIter(TCacheEntryBase* entry);
 
+  // Return all possible overlapping textures. As addr+size of the textures is not
+  // indexed, this may return false positives.
+  std::pair<TexAddrCache::iterator, TexAddrCache::iterator>
+  FindOverlappingTextures(u32 addr, u32 size_in_bytes);
```

Comment:
I think it would be fine:

`std::pair<iter, iter> FindOverlappingTextures(...) const`

Just says that the function will not modify any class members.  So the implicit 'this' pointer is constant.  (this is what I'm suggesting).  It just marks it so the compiler catches it if you try to change something.

`std::pair<iter, iter> const FindOverlappingTextures(...)`

or

`const std::pair<iter, iter> FindOverlappingTextures(...)`

Would just say that the pair itself was const but as it is a pair of iters you could change what the iterators are pointing to.  Still, this wouldn't be good because you want to set the iterator on line 1297 of TextureCacheBase (with the Invalidate call).

`std::pair<const iter, const iter> FindOverlappingTextures(...)`

Would have constant pointers, so you couldn't change what they are pointing to...useful in some scenarios.

Isn't the C++ language great?! :)

Anyway, no biggie either way, just thought it might be a useful addition.